app/models/manifest_uploader.rb

```diff
@@ -76,39 +56,25 @@ def check_locations
     errors.add(:base, "location(s) with barcode #{missing_locations.join(',')} do not exist")
   end
 
-  def check_duplicate_positions
-    location_groups = ordered_location_rows.group_by { |row| [row[1], row[3]] }.values.select { |group| group.length > 1 }
+  # If a location has coordinates it will cause all sorts of problems and should fail
+  # otherwise it will cause all sorts of problems downstream
+  def check_for_ordered_locations
+    ordered_locations = locations.select { |_k, v| v.ordered? }
```

Comment:
There is a comment above but maybe not clear enough. An ordered location is one with coordinates usually it will be a box that you put tubes in. They use them for sequencing libraries. I think there may have been instances where they have mistakenly created an ordered location. We need to make sure that never happens hence the check.